src/content/ru/resources/markdown-syntax.md

```diff
@@ -0,0 +1,617 @@
+работают, и также гарантирует, что любые различия между Markdown процессорами -
+не проблема.
+
+### Заголовок третьего уровня (Heading 3)
+
+DevSite автоматически добавит `<h2>` и `<h3>` тэги к оглавлениям. Чтобы избежать
+добавления (как эти два, которые здесь не отображаются в оглавлении),
+используйте `class="hide-from-toc"`.
+Вы можете также поместить `<h2>` или `<h3>` в заголовки таблиц (`<th>`), чтобы
+таблица появилась в оглавлении. Внутри заголовка таблицы, `<h2>` и `<h3>`
+оформлены как обычный текст, так что читалки не будут их читать.
+
+#### Заголовок четвертого уровня (Heading 4)
+
+##### Заголовок пятого уровня (Heading 5)
+
+###### Заголовок шестого уровня (Heading 6)
+
+```
+## Heading 2
+### Heading 3
+#### Heading 4
+##### Heading 5
+###### Heading 6
+```
+
+## Пример кода
+
+Если ваша статья содержит код, пожалуйста убедитесь, что он соответствует
+[Руководству по стилю JavaScript от
+Google](https://google.github.io/styleguide/javascriptguide.xml).
+Иначе, мы будем вынуждены просить вас поправить недочеты, а это никого не
+радует.
+
+### Встроенный код
+
+Чтобы обозначить строку кода, оберните её в обратные кавычки (`). В отличие от
+преформатированного блока кода, кодовая строка отображается внутри текста
+Например:
+
+```
+Use the `printf()` function.
+```
+
+Результатом будет:
+
+Use the `printf()` function.
+
+### Блоки кода
+
+Чтобы создать блок кода в Markdown, просто сделайте отступ какждой строки кода
+не менее 4 пробелов или 1 таба. Например, дан такой код:
+
+```
+Here is an example of AppleScript:
+
+    tell application "Foo"
+        beep
+    end tell
+```
+
+результатом будет:
+
+Here is an example of AppleScript:
+
+```
+tell application "Foo"
+    beep
+end tell
+```
+
+Внутри блока кода, амперсанды (`&`) и угловые скобки (`<` и `>`) автоматически
+конвертируются в  HTML коды. Это делает очень простым включение примеров HTML
+кода используя Markdown -- просто вставьте их и сделайте отступ, и Markdown
+возьмет на себя хлопоты по кодированию амперсандов и угловых скобок.
+
+Warning: Оборачивание блоков кода в `&amp;&lt;&gt;` не поддерживается DevSite.
+DevSite не стилизует эти блоки автоматически и наши интеграционные тесты
+провалятся.
+
+### Выделение
+
+Используйте `<strong>`, чтобы привлечь внимание к контенту внутри `<pre>` блока.
+Это позволит осветить окружающий контент, чтобы подчеркнуть код внутри блока
+`<strong>`. Например:
+
+<pre class="prettyprint">
+// ...
+// ...
+// ...
+for (i = 0; i if (i % 3 == 0) {
+        someFunc(i);
+    }
+}
+// ...
+// ...
+// ...
+</pre>
+
+```
+<pre class="prettyprint">
+// ...
+// ...
+// ...
+for (i = 0; i < 10; i++) {
+    printf("Counting %d\n", i);
+
+    <strong>if (i % 3 == 0) {
+        someFunc(i);
+    }</strong>
+}
+// ...
+// ...
+// ...
+</pre>
+```
+
+### Копирование по клику
+
+Копирование по клику автоматически доступно для всех блоков с кодом.
+
+### Особый случай: Шаблоны -  {{}}
```

Comment:
@ilyaspiridonov If you look at [the original source code](https://raw.githubusercontent.com/google/WebFundamentals/master/src/content/en/resources/markdown-syntax.md), this section is intentionally escaped as `&#123;&#123;}}`, however GitLocalize decodes it unexpectedly. This causes our test runner to return an error. I'll file an issue for this.

Reply:
https://github.com/gitlocalize/feedback/issues/74